tests/test_editable.py

```diff
@@ -16,7 +18,17 @@
 
 from mesonpy import _editable
 
-from .test_wheel import EXT_SUFFIX
+from .test_wheel import EXT_SUFFIX, NOGIL_BUILD
+
+
+def find_cython_version():
+    _cython_version_str = subprocess.run(['cython', '--version'], check=True,
```

Comment:
It does not matter, but why the leading `_`?

Reply:
Taken over from `_meson_ver_str` in `test_wheel.py`. Agreed the underscore isn't needed, removed now.